### PR TITLE
Add CUDA support

### PR DESCRIPTION
**Pull request by bandi13**
Added in support for AesEncrypt_C in CUDA

### CODE REVIEW

File: scripts/benchmark_compare.sh [hunk 1]
```diff
@@ -0,0 +1,156 @@
+#!/bin/bash
```
Comment (dgarske) [inline scripts/benchmark_compare.sh:1]: You'll need to add this file to scripts/include.am.

File: wolfcrypt/src/port/cuda/README.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+You will need to have the CUDA libraries and toolchains installed to be able to use this. This code was tested with the following:
```
Comment (dgarske) [inline wolfcrypt/src/port/cuda/README.md:1]: Could we expand the documentation? Seems like there should be a bit more here

**Review by dgarske — CHANGES_REQUESTED** [review]

File: scripts/benchmark_compare.sh (hunk 1 above)
Comment (bandi13) [reply scripts/benchmark_compare.sh:1]: Yup. Already done locally. Didn't want to trigger another build

File: wolfcrypt/src/port/cuda/README.md (hunk 2 above)
Comment (bandi13) [reply wolfcrypt/src/port/cuda/README.md:1]: Sure. I'll add my environment setup.

**Comment by bandi13** [comment]
Retest this please

**Review by dgarske — APPROVED** [review]

**Comment by dgarske** [comment]
@night1rider will you plan to take this for a test-drive?

**Pull request merged**